node/Cargo.toml

```diff
@@ -3,103 +3,102 @@ name = "zeitgeist"
+session-keys-primitives = { tag = "v0.27.2-a", default-features = false, git = "https://github.com/zeitgeistpm/moonbeam", optional = true }
+sp-keystore = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate", optional = true }
+substrate-prometheus-endpoint = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate", optional = true }
 
 # Polkadot
 
-polkadot-cli = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-parachain = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-primitives = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-service = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-test-service = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
+polkadot-cli = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-parachain = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-primitives = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-service = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-test-service = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
 
 # Standalone
 
-sc-consensus-aura = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
-sc-finality-grandpa = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
-sp-consensus-aura = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
-sp-finality-grandpa = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
+sc-consensus-aura = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
+sc-finality-grandpa = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
+sp-consensus-aura = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
+sp-finality-grandpa = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
 
 # Utility
 cfg-if = { version = "1.0.0" }
-clap = { version = "3.2.6", features = ["derive"] }
+clap = { version = "3.2.20", features = ["derive"] }
 hex-literal = { version = "0.3.4" }
-jsonrpsee = { version = "0.14.0", features = ["server"] }
+jsonrpsee = { version = "0.15.1", features = ["server"] }
 log = { optional = true, version = "0.4.17" }
-# TODO(#865): Remove in future Polkadot release
-tracing-core = "=0.1.26"
```

Comment:
For what moonbeam uses tracing-core = "0.1.29" https://github.com/PureStake/moonbeam/compare/v0.26.0...v0.27.2#diff-5dc0537169cb810865347de9185dae9ac7d231b7b5eb18eae2086c578dda133eR27 ?

Reply:
I am not sure, we had used explicit version of tracing-core because traces in try-runtime was not displayed properly.
but now with this upgrade we no longer need this and it works fine.